### ISSUE TITLE
InsecurePlatformWarning: A true SSLContext object is not available

### ISSUE THREAD

**Issue by holms** [nephila/djangocms-installer#174]
Tried to setup a project, with latest version from pip.

```
(bandmix)holms@holms:/var/www/bandmix/project$ djangocms -p . bandmix
Database configuration (in URL format) [default sqlite://localhost/project.db]:
django CMS version (choices: 2.4, 3.0, stable, develop) [default stable]:
Django version (choices: 1.4, 1.5, 1.6, 1.7, stable) [default stable]:
Activate Django I18N / L10N setting (choices: yes, no) [default yes]: no
Install and configure reversion support (choices: yes, no) [default yes]: yes
Languages to enable. Option can be provided multiple times, or as a comma separated list. Only language codes supported by Django can be used here:
Languages to enable. Option can be provided multiple times, or as a comma separated list. Only language codes supported by Django can be used here: lt
Optional default time zone [default Etc/UTC]: Europe/Vilnius
Activate Django timezone support (choices: yes, no) [default yes]:
Activate CMS permission management (choices: yes, no) [default yes]:
Use Twitter Bootstrap Theme (choices: yes, no) [default no]: yes
Use custom template set [default no]: yes
Given directory does not exists, retry
Use custom template set [default no]: no
Load a starting page with examples after installation. Choose "no" if you use a custom template set. (choices: yes, no) [default no]: yes
Creating the project
Please wait while I install dependencies
/var/www/bandmix/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:79: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
Dependencies installed
Creating the projectFailure occurred. Do you want to cleanup by removing /var/www/bandmix/project? [Y/N]

```

**Comment by yakky** [comment]
Which python/pip version?

**Comment by holms** [comment]
```
(bandmix)$ python --version
Python 2.7.3
(bandmix)$ pip --version
pip 6.1.1 from /var/www/bandmix/local/lib/python2.7/site-packages (python 2.7)
```

**Comment by holms** [comment]
On default digital ocean droplet of Ubuntu 14.04 x64, if you're interested :)

**Comment by holms** [comment]
Seems to be installing this package from pip resolved an issue, but i'm not sure if this should be like it is.. 
```
requests[security]
```
People suggests to start using urllib3:
http://stackoverflow.com/questions/29134512/insecureplatformwarning-a-true-sslcontext-object-is-not-available-this-prevent

**Comment by yakky** [comment]
It's not due to the requests version (requests is only used in a currently disabled option), but I guess the certifi package that gets installed with it, which pip uses if present

**Comment by hfv1606** [comment]
sudo apt-get install -y libffi-dev libssl-dev
    pip install requests[security]

should this issue for me

Unfortunately I encounter another issue:

EnvironmentError: Pillow is not compiled with JPEG support, see 'Libraries installation issues' documentation section: http://djangocms-installer.readthedocs.org/en/latest/libraries.html

documentation didn't help

**Comment by yakky** [comment]
@hfv1606 What if you pip install pillow yourself? Does it comes with png and jpeg support?

**Comment by hfv1606** [comment]
error "Pillow is not compiled with JPEG support" solved with

sudo apt-get install -y libtiff5-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python-tk

**Comment by yakky** [comment]
Clarify the libraries section with examples. Thanks @hfv1606

**Comment by hfv1606** [comment]
I don't know what you mean by examples

But I will describe my case as well as I can.

I have made a virtual machine for virtual box with vagrant.
I installed Ubuntu 14.04.2 LTS

The commands below lead to a succesfull install of djangocms with djangocms-installer. 

    APPOWNER=holke

    PROJNAME=$APPOWNER
    ENVNAME=$APPOWNER

    TARGETDIR=/home/vagrant
    PROJHOME=$TARGETDIR/proj
    PROJDIR=$PROJHOME/$APPOWNER
    ENVHOME=$TARGETDIR/env

    sudo apt-get install -y libpq-dev python-dev 
    sudo apt-get install -y python-pip

    # django cms: InsecurePlatformWarning: A true SSLContext object is not available
    sudo apt-get install -y libffi-dev libssl-dev

    # django cms: library dependencies for pillow: see https://github.com/python-pillow/Pillow/blob/master/docs/installation.rst
    # http://www.renedohmen.nl/blog/2015/05/install-python-pil-or-pillow-on-ubuntu-14-04-64-bit/
    sudo apt-get install -y libtiff5-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python-tk    


    # install virtualenv
    sudo pip install virtualenv

    # setup virtualenv
    mkdir -p $ENVHOME
    cd $ENVHOME
    virtualenv $ENVNAME


    # activate virtualenv
    . $ENVHOME/$ENVNAME/bin/activate


    pip install requests[security]
    pip install pillow

    pip install djangocms-installer
    
    mkdir -p $PROJDIR/$PROJNAME
    cd $PROJDIR/$PROJNAME
    djangocms -p . holke

    #Database configuration (in URL format) [default sqlite://localhost/project.db]:
    #django CMS version (choices: 2.4, 3.0, 3.1, stable, develop) [default stable]:
    #Django version (choices: 1.4, 1.5, 1.6, 1.7, stable) [default stable]:
    #Activate Django I18N / L10N setting (choices: yes, no) [default yes]:
    #Install and configure reversion support (choices: yes, no) [default yes]:
    #Languages to enable. Option can be provided multiple times, or as a comma separated list. Only language codes supported by Django can be used here: en,nl
    #Optional default time zone [default Europe/Amsterdam]:
    #Activate Django timezone support (choices: yes, no) [default yes]:
    #Activate CMS permission management (choices: yes, no) [default yes]:
    #Use Twitter Bootstrap Theme (choices: yes, no) [default no]: yes
    #Use custom template set [default no]:
    #Load a starting page with examples after installation. Choose "no" if you use a custom template set. (choices: yes, no) [default no]: yes
    
    # test it
    python manage.py runserver 0.0.0.0:8000

**Comment by hfv1606** [comment]
I am happy django-cms is finally working. I tried to install it manually several times with different howto's but never succeded in a 100% working installation. 

I tried to install djangocms by using a requirements file. 
    pip freeze > requirements.txt 
    pip install -r requirements.txt. 

However that leads to:     
    No distributions matching the version for djangocms-text-ckeditor==2.5.0b1

**Comment by yakky** [comment]
@hfv1606 sorry, my comment was not entirely clear.
With the above PR I removed  requests from dependencies, as it's only used in a dead part of the code due to be removed and added examples to the libraries section of the documentation.
pip freeze reported a wrong version number as djangocms-text-ckeditor was installed from master branch. 2.5.0 has now been released so this is fixed too.
Expect a new installer version to be released soon
